server/container/mailbox-adapter/src/main/java/org/apache/james/adapter/mailbox/MailboxManagerManagement.java

```diff
@@ -167,6 +192,13 @@ private void checkMailboxArguments(String namespace, String user, String name) {
         checkString(name, "mailbox name");
     }
 
+	private void checkMailboxArguments(String namespace, String user, String name, String emlpath) {
+		checkString(namespace, "mailbox path namespace");
+		checkString(user, "mailbox path user");
+		checkString(name, "mailbox name");
+		checkString(emlpath, "email file path name");
```

Comment:
I would remove the checkString private method and do:
```
		Preconditions.checkArgument(!Strings.isNullOrEmpty(namespace), "namespace can't be null or empty");
```
using Strings from Guava: https://google.github.io/guava/releases/18.0/api/docs/com/google/common/base/Strings.html